### DISCUSSION TITLE
Some basic questions about the board... (mostly power related)

### DISCUSSION THREAD

**Discussion by vaderag** [darthcloud/BlueRetro#40] [Q&A] [answered]
I was watching your video here: https://www.youtube.com/watch?v=yj_Zbjb2_ms
and it prompted a couple of questions:

You mention a switch between 3v and 5v on one of the consoles, and other consoles that don't have any power at all - if building to the schematics using an ESP32 what concerns do I need be aware of for the power? Or does the ESP32 handle the voltage change?
Does the ESP32 not need powering externally at all times?
In one case it appeared that maybe you have a power connection on one of the adaptors - maybe it's not one I've looked at but didn't see separate power on the adaptors I looked at in the schematics...

Also looks like you had a custom board made, presumably this is just based on ESP32 in different layout?

**Comment by darthcloud** [comment, marked as answer]
The schematics are complete and the wiki pages add extra info if required. Just follow them and it will work.

https://github.com/darthcloud/BlueRetro/wiki/BlueRetro-DIY-Build-Instructions
https://github.com/darthcloud/BlueRetro/wiki/BlueRetro-Cables-Build-Instructions

Only CD-i & JVS require external power.
DIY & DevKit are the same.
